docker/build/installers/install_cyber_deps.sh

```diff
@@ -24,9 +24,6 @@ CURR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 
 TARGET_ARCH="$(uname -m)"
 
-info "Install poco ..."
-bash ${CURR_DIR}/install_poco.sh
-
```

Comment:
Will rebuild the whole docker images to verify that no other packages/modules depends on Poco.

Reply:
I searched the poco tag and found it only in classloader. rebuild the docker will be better.